src/main/java/Airport.java

```diff
@@ -1,55 +1,79 @@
-import Planes.ExperimentalPlane;
-import Planes.MilitaryPlane;
-import Planes.PassengerPlane;
-import Planes.Plane;
 import exceptions.NonMilitaryPlainException;
 import models.MilitaryType;
+import planes.ExperimentalPlane;
+import planes.MilitaryPlane;
+import planes.PassengerPlane;
+import planes.Plane;
 
 import java.util.*;
 
+import static com.sun.xml.internal.ws.spi.db.BindingContextFactory.LOGGER;
+
 
 public class Airport {
     private List<? extends Plane> planes;
 
 
     public List<PassengerPlane> getPassengersPlane() {
-        List<? extends Plane> planesList = this.planes;
         List<PassengerPlane> passengersPlaneList = new ArrayList<>();
-        for (Plane plane : planesList) {
-            if (plane instanceof PassengerPlane) {
-                passengersPlaneList.add((PassengerPlane) plane);
-            }
+        for (Plane plane : planes) {
+            addPlaneIfItIsPassenger(plane, passengersPlaneList);
         }
         return passengersPlaneList;
     }
 
+    public List<PassengerPlane> addPlaneIfItIsPassenger(Plane plane, List<PassengerPlane> passengerPlaneList) {
+        if (plane instanceof PassengerPlane) {
+            passengerPlaneList.add((PassengerPlane) plane);
+        } else {
+            LOGGER.info("Plane is not passenger");
+        }
+        return passengerPlaneList;
+    }
+
+
     public List<MilitaryPlane> getMilitaryPlanes() throws NonMilitaryPlainException {
         List<MilitaryPlane> militaryPlanes = new ArrayList<MilitaryPlane>();
         for (Plane plane : planes) {
-            if (plane instanceof MilitaryPlane) {
-                militaryPlanes.add((MilitaryPlane) plane);
-            } else {
-                throw new NonMilitaryPlainException("Plane is not military");
-            }
+            addPlaneIfItIsMilitary(plane, militaryPlanes);
         }
         return militaryPlanes;
     }
 
-    public PassengerPlane getPassengerPlaneWithMaxPassengersCapacity() {
+    public List<MilitaryPlane> addPlaneIfItIsMilitary(Plane plane, List<MilitaryPlane> militaryPlanes) {
+        if (plane instanceof MilitaryPlane) {
+            militaryPlanes.add((MilitaryPlane) plane);
+        } else {
+            LOGGER.info("Plane is not military");
+        }
+        return militaryPlanes;
+    }
+
+    public PassengerPlane getPassengerPlaneWithMaxPassengersCapacity() throws NonMilitaryPlainException {
         List<PassengerPlane> passengerPlanes = getPassengersPlane();
         PassengerPlane planeWithMaxCapacity = passengerPlanes.get(0);
-        for (int i = 0; i < passengerPlanes.size(); i++) {
+        return findInCyclePlaneWithMaxCapacity(passengerPlanes, planeWithMaxCapacity);
+    }
+
+    public PassengerPlane findInCyclePlaneWithMaxCapacity(List<PassengerPlane> passengerPlanes, PassengerPlane planeWithMaxCapacity) throws NonMilitaryPlainException {
```

Comment:
You first and second method iterates over the same list.
probably let first method iterates, while the second makes comparison logic. And, no need in this variable duplication in the first string.
ex(non-functional, for easier understanding):
public PassengerPlane getPassengerPlaneWithMaxPassengersCapacity() throws NonMilitaryPlainException {
PassengerPlane planeWithMaxCapacity = passengerPlanes.get(0); // but the way, it's not null safe and will throw an exception for empty list. to avoid this it either need to be null checked or use java8 stream.
for (int i = 0; i < int; passengerPlanes.size(); i++) {
planeWithMaxCapacity = (Plane)comparePlanesByCapacity(Plane planeWithMaxCapacity, Plane passengerPlanes.get(i))
}
return planeWithMaxCapacity;
}

private Plane comparePlanesByCapacity(Plane plane1, Plane plane2){ //of course it's even better to add a capacity comparator to the parent Plane class.
plane1.getPassengersCapacity() > plane2.getPassengersCapacity() ? return plane1 : return plane2;
}